packages/sdk/Dockerfile

```diff
@@ -189,5 +190,31 @@ RUN ARCH=$(dpkg --print-architecture || uname -m) && \
     rm /tmp/migrate.deb && \
     migrate --version
 
+# Get migration files for rollups-graphql
+ARG GRAPHQL_REPO=cartesi/rollups-graphql
+ARG GRAPHQL_TAG=v2.3.8
+ARG GRAPHQL_GIT_FOLDER=dev/migrations
+ARG GRAPHQL_DEST_FOLDER=db/rollups-graphql
+RUN git clone --depth 1 --branch "$GRAPHQL_TAG" --filter=blob:none --sparse "https://github.com/$GRAPHQL_REPO.git" && \
+    mkdir -p "$GRAPHQL_DEST_FOLDER" && \
+    cd "$(basename "$GRAPHQL_REPO")" && \
+    git sparse-checkout set "$GRAPHQL_GIT_FOLDER" && \
+    mv "$GRAPHQL_GIT_FOLDER" ../$GRAPHQL_DEST_FOLDER && \
+    cd .. && \
+    rm -rf "$(basename "$GRAPHQL_REPO")"
+
+# Get migration files for rollups-espresso-reader
+ARG ESPRESSO_REPO=cartesi/rollups-espresso-reader
+ARG ESPRESSO_TAG=v0.2.1-node-20250128
+ARG ESPRESSO_GIT_FOLDER=internal/repository/postgres/schema/migrations
+ARG ESPRESSO_DEST_FOLDER=db/rollups-espresso-reader
+RUN git clone --depth 1 --branch "$ESPRESSO_TAG" --filter=blob:none --sparse "https://github.com/$ESPRESSO_REPO.git" && \
+    mkdir -p "$ESPRESSO_DEST_FOLDER" && \
+    cd "$(basename "$ESPRESSO_REPO")" && \
+    git sparse-checkout set "$ESPRESSO_GIT_FOLDER" && \
+    mv "$ESPRESSO_GIT_FOLDER" ../$ESPRESSO_DEST_FOLDER && \
+    cd .. && \
+    rm -rf "$(basename "$ESPRESSO_REPO")"
+
```

Comment:
When you point to the `prerelease/v2-alpha` branch, you'll notice that we have `CARTESI_ROLLUPS_GRAPHQL_VERSION` and `CARTESI_ESPRESSO_READER_VERSION` build args.

I recommend you reuse those.

Ideally, the migration code should be part of a downloadable artifact that won't require us to make a git clone during the build process, but just download it somewhere in the container.

Reply:
I removed the git clone step and replaced it with a direct download of the source files, as the source are around 300k, and I don’t think it’s necessary to zip another file. I’ve also reused the CARTESI_ROLLUPS_GRAPHQL_VERSION and CARTESI_ESPRESSO_READER_VERSION build arguments as suggested